communication/slack-config/sig-network/config.yaml

```
@@ -2,4 +2,3 @@ channels:
   - name: sig-network-gateway-api
     id: CR0H13KGA
   - name: sig-network-policy-api
```

> Deleting channels is not permitted, but a channel can be archived by specifying archived: true, or unarchived by removing it (or specifying false).

ref: https://github.com/kubernetes/community/blob/9b7157ce64a0948de6ddd76ca4f51e760a4138b6/communication/slack-config/README.md#channels

```suggestion
  - name: sig-network-policy-api
  - name: sig-network-kpng
    archived: true
```

---

Awesome thank you!